src/main/java/com/commercetools/sync/inventories/InventorySync.java

```diff
@@ -151,7 +156,11 @@ public InventorySync(@Nonnull final InventorySyncOptions syncOptions) {
             }
         }
         if (!accumulator.isEmpty()) {
-            completableFutures.add(processBatch(accumulator).toCompletableFuture());
+            final CompletableFuture<Void> batchProcessingFuture =
+                processBatch(accumulator, supplyChannelKeyToId)
+                    .exceptionally(exception  -> null)
```

Comment:
so to me it looks like this
if processBatch completes exceptionally you throw SyncProblemException
and here you ignore it - what's the logic behind it?